packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 48.24.0 - July 8, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Cancelling a command execution should not throw errors ([PR #2316](https://github.com/forcedotcom/salesforcedx-vscode/pull/2316))
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add unlocked package support to Org Browser ([PR #2305](https://github.com/forcedotcom/salesforcedx-vscode/pull/2305), [Issue #2204](https://github.com/forcedotcom/salesforcedx-vscode/issues/2204))
+
+- Enhance performance for extension activation ([PR #2324](https://github.com/forcedotcom/salesforcedx-vscode/pull/2324))
```

Comment:
@rcoringrato-sfdc , would you want to add details about which extension...
Enhance performance by removing Salesforce CLI extension checks during activation. Command-level errors handle CLI installation or configuration issues, if any.